lib/js/src/manager/lifecycle/_LifecycleManager.js

```diff
@@ -484,6 +518,16 @@ class _LifecycleManager {
             this._cleanProxy();
         }
 
+        if (!this._didCheckSystemInfo) {
+            this._didCheckSystemInfo = true;
+            const systemInfo = new SystemInfo(registerAppInterfaceResponse.getVehicleType(), registerAppInterfaceResponse.getSystemSoftwareVersion()) ;
```

Comment:
There should be a check before creating the SystemInfo object that either vehicleType or systemSoftwareVersion exist.
```
if (vehicleType || systemSoftwareVersion)
```